bazel/envoy_build_system.bzl

```diff
@@ -1,4 +1,6 @@
 load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library", "py_proto_library")
+load("@com_lyft_protoc_gen_validate//bazel:pgv_proto_library.bzl", "pgv_cc_proto_library")
```

Comment:
Is this used?

Reply:
No, I will fix. But, now you have volunteered yourself as reviewer :)